funflow/src/Control/Funflow/External/Executor.hs

```diff
@@ -93,8 +93,10 @@ execute store td = logError $ do
         }
     mbParams <- lift $ runMaybeT $
       traverse (paramToText convParam) (td ^. tdTask . etParams)
-    mbEnv <- lift $ runMaybeT $
-      traverse (sequence . second (paramToText convParam))  (td ^. tdTask . etEnv)
+    mbEnv <- case td ^. tdTask . etEnv of
+      EnvInherit -> pure Nothing
+      EnvExplicit x -> lift $ runMaybeT $
+        traverse (sequence . second (paramToText convParam)) x
```

Comment:
A `Nothing` here actually indicates an error in `convParam`. As I understand, this error is currently silently ignored and the external task will inherit the global environment. I think it would be better to fail noisily in this case. Similar to `params` below.